development/nautobot_config.py

```diff
@@ -19,8 +24,24 @@
     }
 }
 
+HIDE_RESTRICTED_UI = os.getenv("HIDE_RESTRICTED_UI", False)
+
+SECRET_KEY = os.getenv("NAUTOBOT_SECRET_KEY", "")
+
+#
+# Logging
+#
+
 DEBUG = True
 
+# Django Debug Toolbar
+DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lambda _request: DEBUG and not TESTING}
+
+if "debug_toolbar" not in INSTALLED_APPS:
+    INSTALLED_APPS.append("debug_toolbar")
+if "debug_toolbar.middleware.DebugToolbarMiddleware" not in MIDDLEWARE:
+    MIDDLEWARE.insert(0, "debug_toolbar.middleware.DebugToolbarMiddleware")
+
```

Comment:
Why move this into the middle of the `Logging` section when it isn't logging-related configuration?